### PR TITLE
[ISSUE #5066] Fix only individual queues are consumed in high tps scenarios

### PR DESCRIPTION
**Pull request by guyinyou**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

#5066 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by guyinyou** [comment]
At high tps (and no performance bottleneck), the consumption rate is much lower than the production rate

image

**Comment by guyinyou** [comment]
broker.log as shown below

image

File: common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java [hunk 1]
```diff
@@ -191,7 +191,7 @@ public class BrokerConfig extends BrokerIdentity {
      */
     private long brokerNotActiveTimeoutMillis = 10 * 1000;
 
-    private boolean netWorkFlowController = true;
+    private boolean netWorkFlowController = false;
```
Comment (RongtongJin) [inline common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java:194]: How about renaming enableNetWorkFlowControl？

Comment (guyinyou) [reply common/src/main/java/org/apache/rocketmq/common/BrokerConfig.java]: done

**Review by duhenglucky — DISMISSED** [review]

**Review by RongtongJin — DISMISSED** [review]

**Review by RongtongJin — APPROVED** [review]

**Comment by fuyou001** [comment]
RocketMQ netty highwatermark default values is small.so we can adjustment

**Comment by zhouxinyu** [comment]
Hi, It's not recommended to disable this flag, we can adjust the netty watermark to mitigate this problem.

**Comment by zhouxinyu** [comment]
```
When the broker ignores the pullRequest, the client can only wait for 30s to time out, and the data of the queue cannot be consumed within 30s.
```

Actually, we can add a more intelligent backoff policy to resolve this issue.

**Comment by lizhanhui** [comment]
@RongtongJin Pull request should NOT be merged till all concerns are clarified.

**Pull request merged**